Source/WebCore/platform/mediastream/RealtimeMediaSourceCenter.cpp

```diff
@@ -197,25 +196,6 @@ void RealtimeMediaSourceCenter::triggerDevicesChangedObservers()
     });
 }
 
-void RealtimeMediaSourceCenter::getDisplayMediaDevices(const MediaStreamRequest& request, MediaDeviceHashSalts&& hashSalts, Vector<DeviceInfo>& displayDeviceInfo, String& firstInvalidConstraint)
-{
-    if (!request.videoConstraints.isValid)
-        return;
```

Comment:
This is probably the part that we need to add in UIProcess to reject immediately when { video: false } or equivalent is given to getDisplayMedia.

Reply:
This check was unnecessary, we reject in `MediaDevices::getDisplayMedia` if the constraints are invalid or over constrained.